.github/workflows/check_unit_tests.yml

```diff
@@ -13,50 +12,35 @@ jobs:
 
     services:
       postgres:
-        image: postgres:13  # Use the latest version of PostgreSQL you need
+        image: postgres:13
         ports:
           - 5432:5432
         env:
-          POSTGRES_USER: postgres
           POSTGRES_PASSWORD: pw
-          POSTGRES_DB: testdb
-        options: >-
-          --health-cmd "pg_isready -U testuser" 
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
 
     steps:
-      # Step 1: Check out the repository
       - name: Checkout code
         uses: actions/checkout@v3
 
-      # Step 2: Set up Python
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
           python-version: '3.11'
           cache: 'pip'
 
-      # Step 3: Install dependencies
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install '.[dev]'
 
-      # Step 4: Wait for PostgreSQL to be ready
       - name: Wait for PostgreSQL to be ready
         run: |
-          until pg_isready -h localhost -p 5432 -U postgres; do
-            echo "Waiting for PostgreSQL to be ready..."
-            sleep 1
-          done
+          until pg_isready -U postgres; do sleep 1; done
 
-      # Step 5: Run the tests
       - name: Run unit tests
         env:
           MPI_DB_TYPE: postgres
-          MPI_DBNAME: testdb
+          MPI_DBNAME: postgres
```

Comment:
Will there be any issues with `MPI_DBNAME` and `MPI_USER` env vars having the same value?

Reply:
Nope, in fact thats the default https://hub.docker.com/_/postgres.

> POSTGRES_DB
> 
> This optional environment variable can be used to define a different name for the default database that is created > when the image is first started. If it is not specified, then the value of POSTGRES_USER will be used.

Reply:
I'm going to revert this change for now.  Many, but not all of the tests are hard coding these values and not reading them from the environment.    I'll make some more comprehensive changes in the future, but want to do that in a differently scoped PR.